.github/workflows/build.yml

```diff
@@ -3,9 +3,7 @@ name: Build and Test
 on:
   pull_request_target:
     types:
-      - opened
```

Comment:
I don't think we can change the types here to labeled. We still want it to run on opened, reopened, and synchronized and don't always want labeled to be required

Reply:
yeah. I thought that too. My plan was. to revert this change while adding additional conditionals below.